pkg/epinio/detail/applications.vue

```diff
@@ -149,7 +149,16 @@ export default Vue.extend<Data, any, any, any>({
 
       const arr: any[] = isArray(commits) ? commits : [commits];
 
-      return arr.map(c => GitUtils[this.gitType].normalize.commit(c));
+      return arr.map(c => ({
+        ...GitUtils[this.gitType].normalize.commit(c),
+        availableActions: [{
+          action:  'editFromCommit',
+          label:   this.t('epinio.applications.actions.editFromCommit.label'),
```

Comment:
We might need a better label for this, something to denote the action won't kick anything off straight away.

Something like `Deploy commit`  (subtle difference from the one word actions we have elsewhere). Not sure on that, open to improvements